client/blocks/product-purchase-features-list/google-my-business.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { selectedSite, translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				icon={ <img alt="" src="/calypso/images/illustrations/google-my-business-feature.svg" /> }
+				title={ translate( 'Google My Business' ) }
+				description={ translate(
+					'View how customers find your business and what action they take by connecting to a Google My Business location.'
+				) }
+				buttonText={ translate( 'Connect Google My Business' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Connect to Google My Business' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).